Formula/supervisor.rb

```diff
@@ -26,15 +27,24 @@ def install
 
     virtualenv_install_with_resources
 
-    etc.install buildpath/"supervisor/skel/sample.conf" => "supervisord.ini"
+    etc.install buildpath/"supervisor/skel/sample.conf" => "supervisord.conf"
   end
 
   def post_install
     (var/"run").mkpath
     (var/"log").mkpath
+    old_conf = "#{etc}/supervisord.ini"
+    ln_s "supervisord.conf", old_conf unless File.exist?(old_conf)
   end
 
-  plist_options manual: "supervisord -c #{HOMEBREW_PREFIX}/etc/supervisord.ini"
+  def caveats
+    <<~EOS
+      The default location for supervisor's configuration file is now:
```

Comment:
Note sure we need this if it's the default unless we've moved the existing one (in which case we could output from `post_install` instead).

Reply:
In that case, a better approach might be to just print a `post_install` warning about the new location if there's a config file in the old place? Users would have to move it over anyway, since a service restart would read the sample config and all their supervised services would disappear.

Reply:
Yeh, sounds perfect 🎉